drivers/hmis/app/models/hmis/hud/processors/enrollment_coc_processor.rb

```diff
@@ -0,0 +1,15 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis::Hud::Processors
+  class EnrollmentCocProcessor < Base
+    def process(field, value)
+      attribute_name = hud_name(field)
+      attribute_value = value
+      @processor.enrollment_coc_factory.assign_attributes(attribute_name => attribute_value)
+    end
```

Comment:
It feels like this could be dried up into the base class:
```suggestion
    def factory_method
      :enrollment_coc_factory
    end
    # move the next bit to the base class
    def process(field, value)
      attribute_name = hud_name(field)
      attribute_value = value
      @processor.send(factory_method).assign_attributes(attribute_name => attribute_value)
    end
```

Reply:
Looking through more, this might be one of the exceptions, so maybe move the most common one up.